pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -68,6 +68,9 @@ type ClusterSpec struct {
 	// MasterInternalName is the internal DNS name for the master nodes
 	MasterInternalName string `json:"masterInternalName,omitempty"`
 
+	// AdditionalSANs adds additional Subject Alternate Names to apiserver cert that kops generates
+	AdditionalSANs []string `json:"additionalSans,omitempty"`
```

Comment:
@justinsb et al ... `AdditionalSANs` ... is SANs a common reference.  SSL and TLS are probably one of my weakest areas ... lol